nextflow.config

```diff
@@ -23,6 +23,12 @@ prov {
         wrroc {
             file = "${params.outdir}/ro-crate-metadata.json"
             overwrite = true
+            agent {
+                name = "John Doe"
+                orcid = "https://orcid.org/0000-0000-0000-0000"
+            }
+            license = "https://spdx.org/licenses/Apache-2.0"
```

Comment:
Can we not pull this information automatically from a `LICEN[CS]E.(md|txt)` file?

Reply:
This new output format for the nf-prov plugin creates RO-Crates that follow [this profile](https://www.researchobject.org/workflow-run-crate/profiles/provenance_run_crate/).

Similar to your previous review concerning the agent, the license is added directly to the RO-Crate. The crate will include input and output files, the pipeline configuration, and the pipeline itself. I will ensure that the `ro-crate-metadata.json` annotates all Workflow components (i.e., everything created by the original workflow author) with the license specified in a `LICENSE.(md|txt)` file.